apps/modernization-ui/src/design-system/autocomplete/single/Autocomplete.tsx

```diff
@@ -37,7 +37,7 @@ const Autocomplete = ({
     const suggestionRef = useRef<HTMLUListElement>(null);
     const inputRef = useRef<HTMLInputElement>(null);
 
-    const [entered, setEntered] = useState(value?.name);
+    const [entered, setEntered] = useState(value?.name ?? '');
```

Comment:
question: can you add a comment as to why entered has to be at least an empty string?  I recall removing it because I didn't see a clear reason

Reply:
Added comment to the code, but adding explanation here as well. Setting to undefined causes the following error in the console:

![image](https://github.com/user-attachments/assets/695b2f3a-2e96-4289-bf1b-c45a61212515)